### DISCUSSION TITLE
Subform field type nesting not possible?

### DISCUSSION THREAD

**Discussion by MrMalkio** [joomla/joomla-cms#36044] [General]
Goal:
When using the subform field type I want to be able to add another field in the list that is also a subform, For example, a subform that has 1 list option 1 text field. When creating a subform such as this the only fields that show in the dropdown provided are all previously created custom fields that are not a subform field.

With the advance from repeatable to subform in j4, this is somewhat of an expected feature capability. Am I doing something wrong, might there be more settings I need to change or is it currently just not available?
[Video example ](https://www.awesomescreenshot.com/video/6005943?key=71ee29af4bf68f05b4cb8c9752d3d861&fbclid=IwAR3EanO2qhGgn_bHc27ZfqsKOoK9fiT7VKvRMneQKIKef2gWWywaA4XhxLc)

**Comment by Fedik** [comment]
Subform field supports nesting:
```xml
<field name="subform_parent" label="Subform Parent" type="subform" multiple="true">
    <form>
        <field
                name="test_text"
                label="Test Text"
                type="text"
        />
        <field
                name="child_subform"
                label="Child Subform"
                type="subform"
                multiple="true"
        >
            <form>
                <field
                        name="child_text"
                        label="Child Field"
                        type="text"
                />
            </form>
        </field>
    </form>
</field>
```
However I not sure about Subform in "Custom field"
